src/grpc/enrollment.rs

```diff
@@ -89,21 +90,23 @@ impl EnrollmentServer {
         }
     }
 
-    // check if token provided with request corresponds to a valid enrollment session
+    // check if token provided with request corresponds to a valid session
     async fn validate_session(&self, token: Option<&str>) -> Result<Token, Status> {
+        info!("Start validating session. Token {token:?}");
         let Some(token) = token else {
             error!("Missing authorization header in request");
             return Err(Status::unauthenticated("Missing authorization header"));
         };
-        debug!("Validating enrollment session token: {token}");
+        debug!("Validating session token: {token}");
 
         let enrollment = Token::find_by_id(&self.pool, token).await?;
+        debug!("Verify is token valid {enrollment:?}.");
         if enrollment.is_session_valid(server_config().enrollment_session_timeout.as_secs()) {
-            info!("Enrollment session validated");
+            info!("Session validated");
             Ok(enrollment)
         } else {
-            error!("Enrollment session expired");
-            Err(Status::unauthenticated("Enrollment session expired"))
+            error!("Session expired");
+            Err(Status::unauthenticated("Session expired"))
         }
     }
```

Comment:
Wydaje mi się, że tutaj słowo enrollment było okej, bo to sprawdza sesję enrollmentową